Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-solution-submit/ContestSolutionSubmitPage.tsx

```diff
@@ -344,13 +368,13 @@ const ContestSolutionSubmitPage = () => {
         uploadedFile,
     ]);
 
-    const sumMyPoints = useMemo(() => contest
-        ? contest.problems.reduce((accumulator, problem) => accumulator + problem.points, 0)
-        : 0, [ contest ]);
+    const sumMyPoints = useMemo(() => updatedProblems
+        ? updatedProblems.reduce((accumulator, problem) => accumulator + problem.points, 0)
+        : 0, [ updatedProblems ]);
 
-    const sumAllContestPoints = useMemo(() => contest
-        ? contest.problems.reduce((accumulator, problem) => accumulator + problem.maximumPoints, 0)
-        : 0, [ contest ]);
+    const sumAllContestPoints = useMemo(() => updatedProblems
+        ? updatedProblems.reduce((accumulator, problem) => accumulator + problem.maximumPoints, 0)
+        : 0, [ updatedProblems ]);
```

Comment:
What will happen if `updatedProblems` are not set, but there are problems and the user has points for them, will they be shown? Note: 0 submissions with points for the contest is a valid case (if submissions are deleted).

Reply:
Done.